policy.go

```diff
@@ -33,13 +33,14 @@ const (
 // policy is the interface encapsulating eviction/admission behavior.
 //
 // TODO: remove this interface and just rename defaultPolicy to policy, as we
-// are probably only going to use/implement/maintain one policy.
-type policy interface {
+//
+//	are probably only going to use/implement/maintain one policy.
```

Comment:
the same here as well. This was done by `go fmt` but if you remove the space it won't add the additional line.